boa/src/builtins/json/mod.rs

```diff
@@ -107,7 +107,8 @@ pub fn stringify(_: &mut Value, args: &[Value], ctx: &mut Interpreter) -> Result
     };
     let replacer = match args.get(1) {
         Some(replacer) if replacer.is_object() => replacer,
-        _ => return Ok(Value::from(object.to_json().to_string())),
+        // TODO: handle the error case
+        _ => return Ok(Value::from(object.to_json(ctx).unwrap().to_string())),
```

Comment:
I think we can use `?` instead of `unwrap()` here, and the exception will be propagated.